src/utils/requests.js

```diff
@@ -51,3 +51,14 @@ export const patchArticleVotes = (article_id, votes) => {
       return article;
     });
 };
+
```

Comment:
Nice use of axios, you could use an axios instance, the baseURL functionality would allow you to just provide the start of the API url once in your code rather than repeat it https://notes.northcoders.com/courses/js-front-end/axios#axios-instance